recipes/openmeeg/overwrite_opnmeegpy_installation_path_win.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/wrapping/src/CMakeLists.txt b/wrapping/src/CMakeLists.txt
+index 026606ca..ec0f82df 100644
+--- a/wrapping/src/CMakeLists.txt
++++ b/wrapping/src/CMakeLists.txt
+@@ -62,8 +62,7 @@ execute_process(COMMAND ${PYTHON_EXECUTABLE} "-c" "from distutils.sysconfig impo
+                 RESULT_VARIABLE PYTHON_SITEARCH_NOT_FOUND)
+ #extract correctly python location: e.g lib64/python2.7/site-packages
+ file(RELATIVE_PATH PYTHON_REL_SITE_ARCH ${CMAKE_INSTALL_PREFIX} ${PYTHON_SITE_ARCH})
+-set(PYTHON_SITE_ARCH_ABSOLUTE_PATH "${CMAKE_INSTALL_PREFIX}${PYTHON_REL_SITE_ARCH}")
+-message(FATAL_ERROR "PYTHON_SITE_ARCH_ABSOLUTE_PATH: ${PYTHON_SITE_ARCH_ABSOLUTE_PATH}")
++set(PYTHON_SITE_ARCH_ABSOLUTE_PATH "${CMAKE_INSTALL_PREFIX}/${PYTHON_REL_SITE_ARCH}")
```

Comment:
I don't understand what you are doing here. First you are converting the absolute path to relative in line 8 and then converts relative to absolute path in 11. Why?

Reply:
Sorry that was the wrong patch.